source/docs/articles/local/ssh-tunnels-for-secure-connections-to-pantheon-services.md

```diff
@@ -65,3 +65,11 @@ You can destroy the tunnel by using the port value found within the **Connection
 ```bash
 ps -fU USERNAME | grep "ssh -f" | grep "PORT:" | awk '{print $2}' | xargs kill
 ```
+
+## Troubleshooting
+### Lost Connection to MySQL Server

```

Comment:
Most of the Troubleshooting items start with the error message so the user can find it. Would they know it's an idled database server? Or would they be searching by error message? @rachelwhitton @bmackinney